roles/pgbouncer/config/tasks/main.yml

```
@@ -26,7 +26,7 @@
       become: true
       become_user: postgres
       ansible.builtin.command: >-
-        {{ postgresql_bin_dir }}/psql -p {{ postgresql_port }} -U {{ patroni_superuser_username }} -d postgres -tAXcq
+        {{ postgresql_bin_dir }}/psql -p {{ postgresql_port }} -U {{ patroni_superuser_username }} -d postgres -tAXqc
```

```suggestion
        {{ postgresql_bin_dir }}/psql -p {{ postgresql_port }} -U {{ patroni_superuser_username }} -d postgres -tAXc
```

I think the `-q `option is not required when using -`tAXc`

---

I haven't checked in detail, just tried to make the change least disruptive. 

Nevertheless, I dropped the `-q` option in following commit. Thanks!

---

I haven't checked in detail, just tried to make the change least disruptive.

Anyhow, I've dropped the `-q` option in following update. Thanks!

---

@jimnydev Thanks. Please test it.

---

It took me a bit, but I did test both changes (swapping the options and dropping `q) - both are working fine. Thank you for merging it!